sdk/dotnet/Pulumi/Deployment/Deployment.Runner.cs

```diff
@@ -15,6 +15,15 @@ private class Runner : IRunner
             public Runner(IDeploymentInternal deployment)
                 => _deployment = deployment;
 
+            public Task<int> RunAsync<TStack>() where TStack : Stack, new()
+            {
+                var stack = new TStack();
```

Comment:
can you show what the behavior is if their constructor blows up?

Reply:
Added an extra catch. Now it's
```
     pulumi:pulumi:Stack  azure-play-cs-dev           1 error
Diagnostics:
pulumi:pulumi:Stack (azure-play-cs-dev):
error: Running program 'C:\work\pulumi\azure-play-cs\bin\Debug\netcoreapp3.1\Azure.AppService.dll' failed with an unhandled exception:
System.Reflection.TargetInvocationException: Exception has been thrown by the target of an invocation.
---> System.Exception: Blow up
at MyStack..ctor() in C:\work\pulumi\azure-play-cs\Program.cs:line 226
--- End of inner exception stack trace ---
at System.RuntimeTypeHandle.CreateInstance(RuntimeType type, Boolean publicOnly, Boolean wrapExceptions, Boolean& canBeCached, RuntimeMethodHandleInternal& ctor, Boolean& hasNoDefaultCtor)
at System.RuntimeType.CreateInstanceDefaultCtorSlow(Boolean publicOnly, Boolean wrapExceptions, Boolean fillCache)
at System.RuntimeType.CreateInstanceDefaultCtor(Boolean publicOnly, Boolean skipCheckThis, Boolean fillCache, Boolean wrapExceptions)                                                                         
at System.Activator.CreateInstance[T]()
at Pulumi.Deployment.Runner.RunAsync[TStack]() in C:\Users\shilk\go\src\github.com\pulumi\pulumi\sdk\dotnet\Pulumi\Deployment\Deployment.Runner.cs:line 22
```